src/rates/rates.ts

```diff
@@ -0,0 +1,46 @@
+import Big from "big.js";
+import Asset from "../asset";
+import StaticRates, { ConfigRates } from "./staticRates";
+import TestnetMarketMaker, {
+  BalanceLookups,
+  TestnetMarketMakerConfig
+} from "./testnetMarketMaker";
+
+export interface TradeAmounts {
+  buyAsset: Asset;
+  sellAsset: Asset;
+  buyNominalAmount: Big;
+  sellNominalAmount: Big;
+}
+
+export interface Rates {
```

Comment:
renamed to `TradeEvaluationService` because it does not necessarily deal with rates, but just decides if a trade is acceptable at the moment (other "evaluation" might be added later...)